src/scss/_structure.scss

```diff
@@ -14,12 +14,12 @@ main {
 }
 
 // Horizontal Rules
-$hr-color: $gray-light;
+$hr-color: #ddd;
```

Comment:
Let's move this to Line 1. Keeping variables at the top will make life a bit easier down the road. (And yes, I've probably not been consistent about this myself.)